lib/src/main/java/com/diffplug/spotless/yaml/JacksonYamlStep.java

```diff
@@ -19,67 +19,62 @@
 import java.io.Serializable;
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
-import java.util.Arrays;
-import java.util.List;
 import java.util.Objects;
 
 import com.diffplug.spotless.FormatterFunc;
 import com.diffplug.spotless.FormatterStep;
 import com.diffplug.spotless.JarState;
 import com.diffplug.spotless.Provisioner;
+import com.diffplug.spotless.json.JacksonConfig;
 
 /**
  * Simple YAML formatter which reformats the file according to Jackson YAMLFactory.
  */
 // https://stackoverflow.com/questions/14515994/convert-json-string-to-pretty-print-json-output-using-jackson
-public class YamlJacksonStep {
+public class JacksonYamlStep {
```

Comment:
This breaks public API, but I'm okay with it because I'm sure no one has started depending on this and we're about to break other things too. But it needs to be annotated with `**BREAKING**` in the changelog.